### PR TITLE
Update v-import-cpanel Fix 500 Error from php version

### PR DESCRIPTION
**Pull request by vipxr**
Some users reported 500 Error after migration so I edited the script to check the installed versions on hestia and set the php version based on the backup file if it available

### CODE REVIEW

File: bin/v-import-cpanel [hunk 1]
```diff
@@ -251,6 +251,27 @@ rsync -av --exclude-from='exclude_path' homedir/public_html/ /home/$new_user/web
 chown $new_user:$new_user -R /home/$new_user/web/$main_domain1/public_html
 chmod 751 /home/$new_user/web/$main_domain1/public_html
 
+####### Set Domain PHP version
+PHP_VERSION_LINE=$(grep -r "phpversion:" userdata/$main_domain1)
+CPANEL_PHP_VERSION=${PHP_VERSION_LINE#*: }
+CPANEL_PHP_VERSION=$(echo $CPANEL_PHP_VERSION | grep -oP '(?<=php)\d+')
+HESTIA_PHP_VERSION="PHP-${CPANEL_PHP_VERSION:0:1}_${CPANEL_PHP_VERSION:1}"
+
+if /usr/local/hestia/bin/v-list-web-templates-backend | grep -qw "$HESTIA_PHP_VERSION"; then
```
Comment (jaapmarcus) [inline bin/v-import-cpanel]: Please use:
$BIN

instead of /usr/local/hestia/bin

File: bin/v-import-cpanel [hunk 2]
```diff
@@ -251,6 +251,27 @@ rsync -av --exclude-from='exclude_path' homedir/public_html/ /home/$new_user/web
...
+    echo "Setting PHP version to $HESTIA_PHP_VERSION for $main_domain1 under user $new_user"
+    /usr/local/hestia/bin/v-change-web-domain-backend-tpl $new_user $main_domain1 $HESTIA_PHP_VERSION
```
Comment (jaapmarcus) [inline bin/v-import-cpanel]: Same here

**Review by jaapmarcus — CHANGES_REQUESTED** [review]

**Comment by jaapmarcus** [comment]
Please use $BIN instead the full path 👍🏻

File: bin/v-import-cpanel (hunk 2 above)
Comment (vipxr) [reply bin/v-import-cpanel]: Done, Thank you

File: bin/v-import-cpanel (hunk 1 above)
Comment (vipxr) [reply bin/v-import-cpanel]: Done, Thank you

**Comment by vipxr** [comment]
> Please use $BIN instead the full path 👍🏻

Noted , Thank you  👍🏻

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**